src/sql/src/statements.rs

```diff
@@ -128,6 +129,26 @@ fn parse_string_to_value(
     }
 }
 
+fn parse_hex_string_to_value(s: String) -> Result<Value> {
```

Comment:
What about using the `hex` crate? It provides a [`decode`](https://docs.rs/hex/0.4.3/hex/fn.decode.html) function.

Reply:
it needs `decode` actually. have fixed this.